users/bocaj/process_records.h

```diff
@@ -43,7 +41,9 @@ bool process_record_keymap(uint16_t keycode, keyrecord_t *record);
 #define KC_LWEN LT(_LOWER, KC_ENTER)
 #define KC_ADJS TT(_ADJUST)
 #define KC_NUMS TT(_LOWER)
-#define KC_GAME TT(_DIABLO)
+//#define KC_GAME TT(_DIABLO)
+#define XXXXXXX KC_NO
+#define _______ KC_TRNS
```

Comment:
These shouldn't be needed, as they're defined in quantum.

Reply:
They've been removed. Thanks!